arangod/Cache/Manager.h

```diff
@@ -78,20 +78,32 @@ class Manager {
   typedef std::function<bool(std::function<void()>)> PostFn;
 
  public:
+  struct MemoryStats {
+    std::uint64_t globalLimit = 0;
+    std::uint64_t globalAllocation = 0;
+    std::uint64_t spareAllocation = 0;
+    std::uint64_t activeTables = 0;
+    std::uint64_t spareTables = 0;
+  };
+
   static constexpr std::uint64_t minSize = 1024 * 1024;
```

Comment:
I think, according to code style, this and the one below should start with the letter `k`

Reply:
Changed